14.0 sync remove website dep (#293)
itpp-labs/sync-addons: sync/doc/changelog.rst

`4.0.0`
-------

> **Improvement:** remove website module from dependencies and transfer its functionality to handle webhooks

Reviewer: ```suggestion
**Improvement:** remove Website module from dependencies, handle webhooks directly in Sync Studio
```
